keyboards/lets_split/keymaps/tylerwince/keymap.c

```diff
@@ -0,0 +1,249 @@
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
```

Comment:
Lines 9-14 should be changed to:

```c
enum layer_names {
    _QWERTY,
    _COLEMAK,
    _DVORAK,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```

This will change your layer indexes, as having them non-consecutive will increase your firmware size for no benefit or change in functionality.